### PR TITLE
tests: remove travis

### PR DESCRIPTION
**Pull request by connorjclark**
part of #11194

### CODE REVIEW

File: lighthouse-core/scripts/cleanup-LHR-for-diff.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ function cleanAndFormatLHR(lhrString) {
   // TODO: Resolve the below so we don't need to force it to a boolean value:
   // 1) The string|boolean story for proto
   // 2) Travis gets a absolute path during yarn diff:sample-json
-  lhr.configSettings.auditMode = true;
+  // lhr.configSettings.auditMode = true; // testing...
```
Comment (connorjclark) [inline lighthouse-core/scripts/cleanup-LHR-for-diff.js:32]: seeing what happens in actions ...

Comment (connorjclark) [reply lighthouse-core/scripts/cleanup-LHR-for-diff.js]: ya same issue in actions

**Review by patrickhulce — APPROVED** [review]
remove the "Debugging Travis via docker image" section in hacking-tips too?

**Comment by connorjclark** [comment]
BYE

**Pull request merged**